src/bubble_sort.py

```diff
@@ -0,0 +1,28 @@
+"""Bubble sort algorithm."""
+import timeit
+
+
+def bubble_sort(l):
+    """Bubble Sort: Input list or tuple."""
+    if not isinstance(l, (list, tuple)):
+        raise TypeError('Please input a list or tuple.')
+    n = len(l)
+    swapped = True
+    while swapped:
+        swapped = False
+        for i in range(n - 1):
+            j = i + 1
+            if l[i] > l[j]:
+                temp = l[j]
+                l[j] = l[i]
+                l[i] = temp
+                swapped = True
+    return(l)
+
+
+if __name__ == '__main__':
+    user_input = input('Input comma seperated numbers: ')
```

Comment:
I think you misinterpreted this part of the assignment. I don't want you to take in user input. I want you to show the time it takes to run your algorithm on the best case and the worst case lists.